ivy_tests/test_ivy/helpers/hypothesis_helpers/dtype_helpers.py

```diff
@@ -99,11 +99,53 @@ def get_dtypes(
         returns the complete list of valid types
     none
         allow none in the list of valid types
+    key
+        TODO: add description
```

Comment:
The key parameter is here for `st.shared`, if provided, a shared value will be drawn using `st.shared`